var/spack/repos/builtin/packages/py-langsmith/package.py

```diff
@@ -13,6 +13,11 @@ class PyLangsmith(PythonPackage):
 
     license("MIT")
 
+    version(
+        "0.1.1",
+        sha256="10ff2b977a41e3f6351d1a4239d9bd57af0547aa909e839d2791e16cc197a6f9",
```

Comment:
How was this calculated?  Might it have changed?  I'm getting:

```suggestion
        sha256="09df0c2ca9085105f97a4e4f281b083e312c99d162f3fe2b2d5eefd5c3692e60",
```

Reply:
Interesting.  I can check the sha256 but cannot stage this version of the package to check dependencies.

Reply:
Weird, maybe this replaced it inline.  I used spack checksum to create this.  I'm happy to remove this from the PR.

Reply:
```console
 2% ❯ spack clean -d
 2% ❯ spack checksum py-langsmith 0.1.1
==> Found 1 version of py-langsmith
==> Fetching https://files.pythonhosted.org/packages/12/2a/f091e2d0922469b5a96c33238f66340c507d43298ee0802a20f1c66759d3/langsmith-0.1.1-py3-none-any.whl

    version("0.1.1", sha256="10ff2b977a41e3f6351d1a4239d9bd57af0547aa909e839d2791e16cc197a6f9", expand=False)
```

I'm still getting the same checksum.  I don't know what is going on in your case.

Reply:
```
$ spack checksum py-langsmith  0.1.1
==> Found 1 version of py-langsmith
==> Fetching https://files.pythonhosted.org/packages/source/l/langsmith/langsmith-0.1.1.tar.gz

    version("0.1.1", sha256="09df0c2ca9085105f97a4e4f281b083e312c99d162f3fe2b2d5eefd5c3692e60")
```

Reply:
I have the branch checked out and it is fetching the `.tar.gz` archive.  Looks like yours is fetching the wheel `-py3-none-any.whl`.

Reply:
The sha256 I'm getting also matches the one shown on pypi.org for the version's `view hashes` link at https://pypi.org/project/langsmith/0.1.1/#modal-close.

Reply:
Spack is expecting to use the source not built distribution.

Reply:
I notice there are a number of newer versions, with the latest `0.1.60`.

Reply:
From `pyproject.toml` it looks like`py-pydantic` can be used at v2:

```
[tool.poetry.dependencies]
python = ">=3.8.1,<4.0"
pydantic = ">=1,<3"
requests = "^2"
```

though that spack package doesn't list newer versions.